css/layout.css

```diff
@@ -170,4 +170,59 @@ footer nav a {
 	footer { flex-direction: column; }
 }
 
-@media print { footer, nav { display: none; } }
\ No newline at end of file
+@media print { footer, nav { display: none; } }
+
+#menu-dropdown {
+    background-color: hsla(163, 49%, 14%, .97);
+    position: fixed;
+    top: -100%;
+    left: 0;
+    width: 100%;
+    height: 100%;
+	transform: translateY(-100%);
+    transform-origin: top center;
+    transition: transform .7s ease-in-out;
+    z-index: 999;
+	display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-around;
+	padding: 5rem 0;
+}
+
+#menu-icon:focus + #menu-dropdown, #menu-dropdown:focus-within {
+    top: 0;
+	transform: translateY(0);
+}
+
+#menu-dropdown a {
+	display: flex;
+	position: relative;
+	color: var(--clr-green-darker);
+}
+
+#menu-dropdown a.close-menu {
+    position: absolute;
+    top: 2rem;
+    right: 5rem;
+    width: 30px;
+    height: 30px;
+    background-color: transparent;
+    z-index: 9999;
+    cursor: pointer;
+}
+
+#menu-dropdown a.close-menu::before,
+#menu-dropdown a.close-menu::after {
+    content: '';
+    position: absolute;
+    width: 2px;
+    height: 20px;
+	background-color: var(--clr-green-darker);
+    transform: rotate(45deg);
```

Comment:
I think the transition is can't be seen because of the slide down of the whole menu

Reply:
The mentioned code segment doesn't employ a transition. There, the 'transform' is used to create the 'X' for menu closure. Please let me know if I'm wrong! Thank you for the comments :)